deployment/environment/memory/node.go

```diff
@@ -598,3 +611,54 @@ func (e KeystoreSim) Eth() keystore.Eth {
 func (e KeystoreSim) CSA() keystore.CSA {
 	return e.csa
 }
+
+func setupJD(t *testing.T, app chainlink.Application) {
```

Comment:
Why are we mixing real code and test helpers here...?

Reply:
+1